src/tests/Microsoft.Diagnostics.NETCore.Client/EventPipeSessionTests.cs

```diff
@@ -56,6 +56,8 @@ public void EventPipeSessionStreamTest()
         {
             TestRunner runner = new TestRunner(CommonHelper.GetTraceePathWithArgs(), output);
             runner.Start(3000);
+            output.WriteLine($"[{DateTime.Now.ToString()}] TEST");
+            Thread.Sleep(10_000_000);
```

Comment:
Is this temporary for testing?

Reply:
Yes. Want to make sure the dumps get attached in CI.